### ISSUE TITLE
3.2 cannot compile examples on Linux 

### ISSUE THREAD

**Issue by unbornchikken** [arrayfire/arrayfire#1117]
On Linux Mint 17.2 compiling examples I get:

```
/opt/arrayfire-3/lib/libaf.so.3.2.0: undefined reference to `dlsym'
/opt/arrayfire-3/lib/libaf.so.3.2.0: undefined reference to `dlopen'
/opt/arrayfire-3/lib/libaf.so.3.2.0: undefined reference to `dlclose'
```

**Comment by shehzan10** [comment]
This is a known issue. Please follow https://github.com/arrayfire/arrayfire/pull/1104/files

**Comment by pavanky** [comment]
@unbornchikken This is fixed in the `hotfixes-3.2.1` branch.

Check the following commit: 
https://github.com/arrayfire/arrayfire/commit/ca0c7cc10ba044887a44b6b900da03311fb1d453
